deploy/crds/logging.openshift.io_clusterlogforwarders_crd.yaml

```diff
@@ -0,0 +1,203 @@
+                      mistakes.
+                    type: boolean
+                  name:
+                    description: Name used to refer to the output from a `pipeline`.
+                    type: string
+                  reconnect:
+                    description: Reconnect configures how the output handles connection
+                      failures. Auto-reconnect is enabled by default.
+                    properties:
+                      firstDelayMilliseconds:
+                        description: FirstDelayMilliseconds is the time to wait after
+                          a disconnect before the first reconnect attempt. If reconnect
+                          fails, the delay is doubled on each subsequent attempt.
+                          The default is determined by the output type.
+                        format: int64
+                        type: integer
+                      maxDelayMilliseconds:
+                        description: MaxDelaySeconds is the maximum delay between
+                          failed re-connect attempts, and also the maximum time to
+                          wait for an unresponsive connection attempt. The default
+                          is determined by the output type.
+                        format: int64
+                        type: integer
+                      reliability:
+                        description: Reliability policy for data delivery after a
+                          re-connect.  This is simple short-hand for configuring the
+                          output to a given level of reliability.  The exact meaning
+                          depends on the output `type`.  The default is determined
+                          by the output type.
+                        enum:
+                        - Unreliable
+                        - Retry
+                        type: string
+                    type: object
+                  secret:
+                    description: "SecretRef refers to a `secret` object for secure
+                      communication. \n Client-authenticated TLS is enabled if the
+                      secret contains keys `tls.crt`, `tls.key` and `ca.crt`. Output
+                      types with password authentication will use keys `password`
+                      and `username`, not the exposed 'username@password' part of
+                      the `destination` URL."
+                    properties:
+                      name:
+                        description: Name is unique within a namespace to reference
+                          a secret resource.
+                        type: string
+                      namespace:
+                        description: Namespace defines the space within which the
+                          secret name must be unique.
+                        type: string
+                    type: object
+                  syslog:
```

Comment:
@ewolinetz any elasticsearch or forward specific things we need in the initial release?

Reply:
I don't believe so unless we know for sure there is something specific we _need_ to be able to configure out of the gate